cfme/infrastructure/provider/scvmm.py

```diff
@@ -40,6 +40,8 @@ class SCVMMProvider(InfraProvider):
     settings_key = 'ems_scvmm'
     ui_prov_type = 'Microsoft System Center VMM'
     log_name = 'scvmm'
+    provisioning_dialog_widget_names = (InfraProvider.provisioning_dialog_widget_names
+        - {'environment'} | {'customize'})
```

Comment:
Just curious but how is this different from `- {'environment', 'customize'}`?

Reply:
At first I was like WFT have I done here but it is different. So perhaps I should rephrase this bit a bit to make it more clear for the reader.

The problem is the operator priority. The result of
```
(SOME_SET  - {'environment'} | {'customize'})
```
does always include the 'customize' while result of
```
(SOME_SET  - {'environment', 'customize'})
```
never does.

Should I change this to
```
((SOME_SET  - {'environment'}) | {'customize'})
```
pattern?

Or rather
```
SOME_SET.difference({'environment'}).union('{customize'})
```
or
```
newset = set(SOME_SET)
newset -= {'environment'}
newset |= '{customize'}
```

What do you guys find more clear?

Reply:
For the record, I'd say `SOME_SET.difference({'environment'}).union('{customize'})` is the most clear, and is consistent with your use of `union` and `difference` in the other providers.

That said, the logic is sound, and the clarity is subjective, not going to block the PR on it.

Reply:
@mshriver Ok. I will try keeping this in mind and use this form.